http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/DiskServiceInterp.scala

```diff
@@ -192,6 +200,16 @@ class DiskServiceInterp[F[_]: Parallel](config: PersistentDiskConfig,
   ): F[Vector[ListPersistentDiskResponse]] =
     for {
       ctx <- as.ask
+
+      // throw 403 if user doesn't have project permission
+      hasProjectPermission <- cloudContext.traverse(cc =>
+        authProvider.isUserProjectReader(
+          cc,
+          userInfo
+        )
+      )
+      _ <- F.raiseWhen(!hasProjectPermission.getOrElse(true))(ForbiddenError(userInfo.userEmail, Some(ctx.traceId)))
+
       paramMap <- F.fromEither(processListParameters(params))
```

Comment:
This is very much existing, but why is this called a `paramMap`? it's a tuple

Reply:
I'd guess because it contains a labelMap
<img width="435" alt="image" src="https://github.com/DataBiosphere/leonardo/assets/85642387/872c5121-9fbb-4fb2-8925-2b874f28226c">

Reply:
a bit misleading😢

Reply:
I agree there, lots of misleading names in Leo lol